### PR TITLE
Support creation of business entity

### PR DESCRIPTION
**Pull request by nciemniak**
Add module for interacting with [Business Entities](https://docs.justifi.tech/api-spec#tag/Business), due to upcoming deprecation of using `sub_account_id` in the [Hosted Onboarding](https://docs.justifi.tech/api-spec#tag/Hosted-Onboarding) flow. Will resolve issue https://github.com/justifi-tech/justifi-ruby/issues/48.

Module methods mirror the available API methods for the Business entity (create, list, get, update).

Type of Change
--------------

* [x] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (would cause existing functionality to not work as expected)
* [x] This change requires a documentation update
* [ ] This change is in scope for PCI
* [ ] This requires approval from UX
* [ ] This requires approval from Marketing

Steps for Testing/QA
--------------------
Test the following methods on the `Jusifi::Business` module and ensure successful HTTP response:
* `Justifi::Business.create(params: create_params)`
* `Justifi::Business.get(business_id: business_id)`
* `Justifi::Business.list`
* `Justifi::Business.update(business_id: business_id, params: update_params)`

### CODE REVIEW

**Review by pete2786 — APPROVED** [review]

**Comment by pete2786** [comment]
Thanks so much @nciemniak. I will verify the changes and get a new release out soon.

File: spec/stubs/business.rb [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+            id: "biz_xyz",
+            platform_account_id: "acc_xyz",
+            legal_name: "Business Name",
+            website_url: "https://justifi.ai",
+            email: "business@justifi.ai",
+            phone: "6124011111",
+            doing_business_as: "Best Business",
+            business_type: "for_profit",
+            business_structure: "sole_proprietorship",
+            classification: "government limited non_profit partnership corporation public_company sole_proprietor",
```
Comment (wlads) [inline spec/stubs/business.rb:18]: the `classification` is a combination of `business_type` and `business_structure`, so in this case the correct value should be `sole_proprietor`

```suggestion
            classification: "sole_proprietor",
```

File: spec/stubs/business.rb [hunk 2]
```diff
@@ -0,0 +1,471 @@
...
```
Comment (wlads) [inline spec/stubs/business.rb:245]: same here:

```suggestion
            classification: "sole_proprietor",
```

Comment (nciemniak) [reply spec/stubs/business.rb]: Thanks @wlads, fixed now 👍

**Review by wlads — APPROVED** [review]
LGTM

**Pull request merged**